Doc/reference/simple_stmts.rst

```diff
@@ -966,25 +966,14 @@ The :keyword:`!global` statement
 .. productionlist:: python-grammar
    global_stmt: "global" `identifier` ("," `identifier`)*
 
-The :keyword:`global` statement is a declaration which holds for the entire
-current code block.  It means that the listed identifiers are to be interpreted
-as globals.  It would be impossible to assign to a global variable without
+The :keyword:`global` causes the listed identifiers to be interpreted
```

Comment:
This reads `The global`, so is missing a noun: `The global keyword`

Reply:
Hey @merwok , thanks for the correction. I opened #126720 that reflect this feedback and handles some more mishaps.